.ci/skipped_notebooks.yml

```diff
@@ -558,4 +558,11 @@
 - notebook: notebooks/multimodal-rag/multimodal-rag-llamaindex.ipynb
   skips:
     - os:
-        - macos-13
\ No newline at end of file
+        - macos-13
+- notebook: notebooks/llm-rag-langchain/llm-rag-langchain-genai.ipynb
```

Comment:
what the reason of disabling tests? is it not possible to run langchain with some small model?